client/src/components/RightSidebar/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import MuiAlert from "@material-ui/lab/Alert";
+import { Button, Snackbar } from "@material-ui/core";
+import Cookies from "js-cookie";
+import "./index.css";
+import ConfigureWebhook from "../ConfigureWebhook";
+import { githubPrivateRepoAccessClientID } from "../../utils/constants";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export default function RightSidebar(props) {
+  const [openWebhookDialog, setOpenWebhookDialog] = useState(false);
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+  const [snackbarSeverity, setSnackbarSeverity] = useState("success");
+  const [snackbarText, setSnackbarText] = useState("");
+
+  const handleClickConfigureWebhooks = async () => {
+    if (!Cookies.get("gh_private_repo_token")) {
+      document.getElementById("webhook-scope-link").click();
+    } else {
+      setOpenWebhookDialog(true);
+    }
+  };
+
+  const setSnackbar = (snackbarSeverity, snackbarText) => {
```

Comment:
Since this is a repeating code, we must make a component called snackbar, so that by just passing a few props we can use it. For now, we can ignore this but can improve it in the next refactoring after demo.